tests/travis/prepare_functional_parallel.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+set -e
+trap '>&2 echo Error: Command \`$BASH_COMMAND\` on line $LINENO failed with exit code $?' ERR
+
+test_set_list=$(grep "php72" -Rl src/Test/Functional/Acceptance | sort)
+test_set_count=$(printf "$test_set_list" | wc -l)
+let "test_set_count++"
+
+test_set_size[1]=$(printf "%.0f" $(echo "$test_set_count*0.33" | bc))
+test_set_size[2]=$(printf "%.0f" $(echo "$test_set_count*0.33" | bc))
+test_set_size[3]=$((test_set_count-test_set_size[1]-test_set_size[2]))
+echo "Total = ${test_set_count}; Batch #1 = ${test_set_size[1]}; Batch #2 = ${test_set_size[2]}; Batch #3 = ${test_set_size[3]};";
+
+cp codeception.dist.yml codeception.yml
+echo "groups:" >> codeception.yml
+echo "  php72parallel_*: tests/functional/_data/php72parallel_*" >> codeception.yml
+
+
```

Comment:
Remove the empty line in next task

Reply:
I'll remove it in the scope of the next PR.